src/webpack.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { IgnorePlugin } from 'webpack';
+
+import { REACT013, REACT155 } from './version';
+
+// eslint-disable-next-line import/prefer-default-export
+export function getPluginsForInstalledReact() {
```

Comment:
Do we have any future plans for any other exports from this entry point, such that this shouldn't be the default export?